diff --git a/web/src/compose_reply.ts b/web/src/compose_reply.ts
index 4dd078e1879a5..83a79f63dfd0a 100644
--- a/web/src/compose_reply.ts
+++ b/web/src/compose_reply.ts
@@ -290,6 +290,7 @@ export function quote_message(opts: {
         //     ```quote
         //     message content
         //     ```
+        // Keep syntax in sync with zerver/lib/reminders.py
         let content = $t(
             {defaultMessage: "{username} [said]({link_to_message}):"},
             {
